src/overlays/actors/ovl_En_Heishi/z_en_heishi.h

```diff
@@ -7,13 +7,48 @@ struct EnHeishi;
 
 typedef void (*EnHeishiActionFunc)(struct EnHeishi*, GlobalContext*);
 
+typedef enum {
+    /*  0 */ HEISHI_LIMB_NONE,
+    /*  1 */ HEISHI_LIMB_ROOT,
+    /*  2 */ HEISHI_LIMB_PELVIS,
+    /*  3 */ HEISHI_LIMB_LEFT_THIGH,
+    /*  4 */ HEISHI_LIMB_LEFT_CALF,
+    /*  5 */ HEISHI_LIMB_LEFT_FOOT,
+    /*  6 */ HEISHI_LIMB_RIGHT_THIGH,
+    /*  7 */ HEISHI_LIMB_RIGHT_CALF,
+    /*  8 */ HEISHI_LIMB_RIGHT_FOOT,
+    /*  9 */ HEISHI_LIMB_TORSO,
+    /* 10 */ HEISHI_LIMB_LEFT_SHOULDER,
+    /* 11 */ HEISHI_LIMB_LEFT_FOREARM,
+    /* 12 */ HEISHI_LIMB_LEFT_HAND,
+    /* 13 */ HEISHI_LIMB_RIGHT_SHOULDER,
+    /* 14 */ HEISHI_LIMB_RIGHT_FOREARM,
+    /* 15 */ HEISHI_LIMB_RIGHT_HAND_WITH_SPEAR,
+    /* 16 */ HEISHI_LIMB_HEAD,
+    /* 17 */ HEISHI_LIMB_MAX
+} EnHeishiLimbs;
+
 typedef struct EnHeishi {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x110];
+    /* 0x0144 */ SkelAnime skelAnime;
+    /* 0x0188 */ Vec3s jointTable[HEISHI_LIMB_MAX];
+    /* 0x01EE */ Vec3s morphTable[HEISHI_LIMB_MAX];
     /* 0x0254 */ EnHeishiActionFunc actionFunc;
-    /* 0x0258 */ char unk_258[0x78];
+    /* 0x0258 */ s16 headRotY;
+    /* 0x025A */ s16 headRotX;
+    /* 0x025C */ s16 headRotZ;
+    /* 0x025E */ s16 headRotYTarget;
+    /* 0x0260 */ s16 headRotXTarget;
+    /* 0x0262 */ s16 headRotZTarget;
+    /* 0x0264 */ s32 animIndex;
+    /* 0x0268 */ s32 paramCopy;
```

Comment:
```suggestion
    /* 0x0268 */ s32 paramsCopy;
```
Should be params as it is a copy of the actor field `params`